From 98df6fdc574a85c1cbf467e14dd530a4ac1fba71 Mon Sep 17 00:00:00 2001
From: Jeroen Hofstee <jhofstee@victronenergy.com>
Date: Wed, 25 Jun 2014 17:26:11 +0200
Subject: [PATCH] bla

---
 acinclude.m4 | 7 +------
 1 file changed, 1 insertion(+), 6 deletions(-)

diff --git a/acinclude.m4 b/acinclude.m4
index 6e1ff28..399fd57 100644
--- a/acinclude.m4
+++ b/acinclude.m4
@@ -2533,12 +2533,7 @@ dnl
 AC_DEFUN([PHP_SETUP_LIBXML], [
 AC_CACHE_CHECK([for xml2-config path], ac_cv_php_xml2_config_path,
 [
-  for i in $PHP_LIBXML_DIR /usr/local /usr; do
-    if test -x "$i/bin/xml2-config"; then
-      ac_cv_php_xml2_config_path="$i/bin/xml2-config"
-      break
-    fi
-  done
+       ac_cv_php_xml2_config_path="$PHP_LIBXML_DIR/xml2-config"
 ])
 
   if test -x "$ac_cv_php_xml2_config_path"; then
-- 
1.8.3.2
